.github/workflows/end-to-end-tests.yml

```diff
@@ -61,23 +60,21 @@ jobs:
       - name: Checkout repository
         uses: actions/checkout@93ea575cb5d8a053eaa0ac8fa3b40d7e05a33cc8 # v3.1.0
 
+      - name: Install NodeJS
+        uses: actions/setup-node@8c91899e586c5b171469028077307d293428b516 # v3.5.1
+        with:
+          node-version-file: '.nvmrc'
+          cache: npm
+
       - name: Log debug information
         run: |
           npm --version
           node --version
           curl --version
           git --version
           svn --version
-          php --version
-          php -i
```

Comment:
This workflow doesn't use `setup-php`, so logging the PHP information here does seem useful. Then again, there is also the Docker debug info (on line 91) which is probably more pertinent, so yeah, let's remove this here as it could well confuse people if the PHP version mentioned here vs the one mentioned in the Docker debug is different.

(i.e. no action needed, just documenting the thought process here for future reference)